wordpress-mobile/WordPress-Android
# Plugins remove dialog fix and fluxc hash update

Fixes #6981. This PR adds a newline to the remove plugin dialog and reflects the changes in FluxC for handling remove plugin error gracefully. Since we try to disable the plugin first, if the plugin doesn't exist we might get an unknown plugin error in `onSitePluginConfigured` while removing the plugin which we have to handle it in the client.

To test:
1. Install "Widget Importer & Exporter" on your site (or any other plugin with & in it's name)
2. Go into plugins list and make sure it's name is displayed correctly
3. Go into plugin detail page for any plugin that you can remove (non-Jetpack mostly)
4. Make sure the plugin is active so a dispatch for deactivation will occur
5. Remove the plugin from remote (using a browser)
6. Try to remove the plugin in client and make sure it's handled gracefully (no error should pop up)

/cc @nbradbury
